app/views/notes/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class='container'>
+  <section>
+    <div class="container">
+        <div class="row">
```

Comment:
still does not seems to be correct.

Reply:
![Screenshot from 2022-02-22 12-39-16](https://user-images.githubusercontent.com/82970871/155084833-59b703d9-dad9-4761-8cc1-cdcbd0d1b9dd.png)

Reply:
done